app/views/case_logs/index.html.erb

```diff
@@ -1,16 +1,18 @@
 <% item_label = @pagy.count > 1 ? "logs" : "log" %>
-
 <% if @searched.present? %>
-  <% title = "Your organisation (#{@pagy.count} #{item_label} matching ‘#{@searched}’ of #{@total_count} total logs)" %>
+  <% title = "#{current_user.support? ? 'Logs' : current_user.organisation.name} (#{@pagy.count} #{item_label} matching ‘#{@searched}’)" %>
 <% else %>
-  <% title = "Your organisation (Logs)" %>
+  <% title = current_user.support? ? "Logs" : "#{current_user.organisation.name} (Logs)" %>
```

Comment:
This isn’t quite right.

If support user viewing all logs:

Logs

If support user viewing logs for an organisation:

[[Organisation name]] - Logs

Otherwise:

Logs